views/PrestaShop/Admin/Common/Grid/grid_panel.html.twig

```diff
@@ -16,18 +16,27 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
+{% set displayName = true %}
+{% if (grid.view_options.display_name is defined) %}
+    {% set displayName = grid.view_options.display_name %}
+{% endif %}
+
 <div class="card js-grid-panel" id="{{ grid.id }}_grid_panel" data-hook-name="{{ grid.name|split(' ')|first }}">
   {% block grid_panel_header %}
-    <div class="card-header js-grid-header">
-      <h3 class="d-inline-block card-header-title">
-        {{ grid.name }} ({{ grid.data.records_total }})
-      </h3>
-      {% block grid_actions_block %}
-        <div class="d-inline-block float-right">
-          {{ include('@PrestaShop/Admin/Common/Grid/Blocks/grid_actions.html.twig', {'grid': grid}) }}
-        </div>
-      {% endblock %}
-    </div>
+      {% if displayName is not sameas(false) %}
+      <div class="card-header js-grid-header">
+          {% if displayName is not sameas(false) %}
+          <h3 class="d-inline-block card-header-title">
+            {{ grid.name }} ({{ grid.data.records_total }})
+          </h3>
+        {% endif %}
```

Comment:
You're already in the same `if` statement.
```suggestion
          <h3 class="d-inline-block card-header-title">
            {{ grid.name }} ({{ grid.data.records_total }})
          </h3>
```

Reply:
Ping @sowbiba ;)